.github/workflows/website.yaml

```diff
@@ -74,8 +74,37 @@ jobs:
         with:
           path: ./website/public
 
-      # Commit all changed files back to the repository
-      # - uses: stefanzweifel/git-auto-commit-action@v5
+  # Playwright job
+  playwright:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          submodules: recursive
+
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: github-pages
+          path: ./website/public
+
+      - name: Install Playwright
+        run: |
+          pip install playwright
+          playwright install
+
+      - name: Run Playwright
+        run: |
+          python -c "from playwright.sync_api import sync_playwright; p = sync_playwright().__enter__(); b = p.chromium.launch(); c = b.new_context(); page = c.new_page(); page.goto('file://./website/public/index.html'); page.screenshot(path='screenshot.png'); b.close(); p.__exit__(None, None, None)"
```

Comment:
this is not maintainable, we should use the typescript package and a dedicated file to define it